OneSignalExample/Assets/OneSignal/CHANGELOG.md

```diff
@@ -5,9 +5,95 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-## [3.0.11]
```

Comment:
I think this should be included from the main branch

Reply:
Ok I just added it back there. Probably got deleted using the auto resolver.